.github/workflows/code_style.yml

```diff
@@ -24,20 +24,19 @@ jobs:
           cmake ..
 
       - name: Check code style
-        run: make style-check
-        working-directory: build
+        run: cmake --build build --target clang_format_check_all
 
       - name: Create code style diff
         if: failure()
         run: |
-          ngraph/maint/apply-code-format.sh
-          git diff >ngraph_code_style_diff.patch
+          cmake --build build --target clang_format_fix_all
+          git diff > code_style_diff.patch
```

Comment:
I'd suggest to use `.diff` file extention. A patch is an output of `git format-patch` command and adds additional meta-data to _regular_ diff output.